app/views/partials/_global_nav.html.haml

```diff
@@ -30,12 +30,20 @@
         %ul.nav.navbar-right.navbar-nav
           - if current_user.admin?
             %li= link_to 'Users', new_admin_user_path
+            %li.dropdown
+              %a.dropdown-toggle{"data-toggle" => "dropdown", href: "#"} Supplies
```

Comment:
Nitpick alert: is there a specific reason there is `%a` used here and `= link_to` used elsewhere? If it is because of it not actually linking anywhere, I would consider making this a `%button`, instead.